mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -4217,6 +4480,192 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
             loc, transformedMemRefType,
             inGNCYTildaSliceHTidaSliceXTildaSliceWTildaSlice, transformedAttrs,
             /*populateBounds=*/true);
+
+        // with padding kernel
```

Comment:
same here, please consider if logic here could be extracted to a function on a separate header file.